src/components/PatientsTable/PatientsTable.tsx

```diff
@@ -0,0 +1,59 @@
+//
+// This source file is part of the Stanford Biodesign Digital Health Spezi Web Health Components open-source project
+//
+// SPDX-FileCopyrightText: 2024 Stanford University and the project authors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+import { type Patient } from '@medplum/fhirtypes'
+import {
+  DataTable,
+  type DataTableProps,
+} from '@stanfordspezi/spezi-web-design-system/components/DataTable'
+import { createColumnHelper } from '@tanstack/table-core'
+import { PatientMenu } from './PatientMenu'
+
+interface PatientsDataTableProps
+  extends Omit<DataTableProps<Patient>, 'columns'> {
+  onRowClick?: (data: Patient) => void
+  onDelete: (data: Patient) => void
+  editRoute: string
```

Comment:
I think it would make sense to create some health-components-wide context that gathers all the routes. Routes might be different per application usage, but they're constant. It seems like it makes sense to inject them in just one place and consume that in the components.